### PR TITLE
docs: fix ui and demo directive descriptions

### PR DESCRIPTION
**Pull request by MichaelDemeyer**
Fix small typo in documentation and adjust linter configuration to allow long lines in tables

### CODE REVIEW

File: docs/hub/config.md [hunk 1]
```diff
@@ -175,7 +181,7 @@ When using environment variables, list must be space separated. As flags paramet
...
-| `use_forwarded_headers`    | use the `X-Forwarded-For`, and `X-Real-IP` for the remote (client) IP address, `X-Forwarded-Proto` or `X-Forwarded-Scheme` for the scheme (http or https), `X-Forwarded-Host` for the host and the RFC 7239 `Forwarded` header, which may include both client IPs and schemes. If this option is enabled, the reverse proxy must override or remove these headers or you will be at risk | `false`                                                  |
+| `use_forwarded_headers`    | use the `X-Forwarded-For`, and `X-Real-IP` for the remote (client) IP address, `X-Forwarded-Proto` or `X-Forwarded-Scheme` for the scheme (HTTP or HTTPS), `X-Forwarded-Host` for the host and the RFC 7239 `Forwarded` header, which may include both client IPs and schemes. If this option is enabled, the reverse proxy must override or remove these headers or you will be at risk | `false`                                                  |
```
Comment (dunglas) [inline docs/hub/config.md:184]: this should be reverted, the scheme will be in lowercase (maybe could we put the value into backticks): https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto

File: docs/hub/config.md [hunk 2]
```diff
@@ -91,32 +91,37 @@ Here is an example of how to use environments variables with a RSA public key.
...
+```
 
 Start the hub:
 
+```console
     MERCURE_PUBLISHER_JWT_KEY=$(cat publisher.key.pub) \
     MERCURE_PUBLISHER_JWT_ALG=RS256 \
     MERCURE_SUBSCRIBER_JWT_KEY=$(cat subscriber.key.pub) \
     MERCURE_SUBSCRIBER_JWT_ALG=RS256 \
     ./mercure run
```
Comment (dunglas) [inline docs/hub/config.md:109]:
```suggestion
MERCURE_PUBLISHER_JWT_KEY=$(cat publisher.key.pub) \
MERCURE_PUBLISHER_JWT_ALG=RS256 \
MERCURE_SUBSCRIBER_JWT_KEY=$(cat subscriber.key.pub) \
MERCURE_SUBSCRIBER_JWT_ALG=RS256 \
./mercure run
```

File: docs/hub/config.md [hunk 3]
```diff
@@ -91,32 +91,37 @@ Here is an example of how to use environments variables with a RSA public key.
 
 Generate keys (if you don't already have them):
 
+```console
     ssh-keygen -t rsa -b 4096 -m PEM -f publisher.key
     openssl rsa -in publisher.key -pubout -outform PEM -out publisher.key.pub
     
     ssh-keygen -t rsa -b 4096 -m PEM -f subscriber.key
     openssl rsa -in subscriber.key -pubout -outform PEM -out subscriber.key.pub
```
Comment (dunglas) [inline docs/hub/config.md:99]:
```suggestion
ssh-keygen -t rsa -b 4096 -m PEM -f publisher.key
openssl rsa -in publisher.key -pubout -outform PEM -out publisher.key.pub
    
ssh-keygen -t rsa -b 4096 -m PEM -f subscriber.key
openssl rsa -in subscriber.key -pubout -outform PEM -out subscriber.key.pub
```

File: docs/hub/config.md [hunk 4]
```diff
@@ -91,32 +91,37 @@ Here is an example of how to use environments variables with a RSA public key.
...
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `bucket_name`       | name of the bolt bucket to store events. default to `updates`                                                                                                       |
 | `cleanup_frequency` | chances to trigger history cleanup when an update occurs, must be a number between `0` (never cleanup) and `1` (cleanup after every publication), default to `0.3`. |
-| `size`              | size of the history (to retrieve lost messages using the `Last-Event-ID` header), set to `0` to never remove old events (default)                                                |
+| `size`              | size of the history (to retrieve lost messages using the `Last-Event-ID` header), set to `0` to never remove old events (default)                                   |
 
 Below are common examples of valid DSNs showing a combination of available values:
 
+```Caddyfile
```
Comment (dunglas) [inline docs/hub/config.md:124]: The spaces must be removed then.

**Review by dunglas — CHANGES_REQUESTED** [review]
Good catch, thanks for the fixes!

File: docs/hub/config.md (hunk 1 above)
Comment (MichaelDemeyer) [reply docs/hub/config.md]: Done

File: docs/hub/config.md (hunk 2 above)
Comment (MichaelDemeyer) [reply docs/hub/config.md]: Done

File: docs/hub/config.md (hunk 4 above)
Comment (MichaelDemeyer) [reply docs/hub/config.md:124]: Done

File: docs/hub/config.md (hunk 3 above)
Comment (MichaelDemeyer) [reply docs/hub/config.md]: Done

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks!

**Pull request merged**